File: Frontend/trackr-paciente/src/app/shared/Dtos/perfil/informacion-domicilio-dto.ts
export class InformacionDomicilioDto {
    idPais: number | null;
    idEstado: number | null;
    idMunicipio: number | null;
    idLocalidad: number | null;
    idColonia: number | null;
    codigoPostal: string;
    calle: string;
    entreCalles: string;
    numeroInterior: string;
    numeroExterior: string;
}

File: Backend/TrackrAPI/TrackrAPI/Dtos/Perfil/InformacionDomicilioDTO.cs
using TrackrAPI.Dtos.GestionExpediente;

namespace TrackrAPI.Dtos.Perfil
{
    public class InformacionDomicilioDTO
    {
        public int? IdPais { get; set; }
        public int? IdEstado { get; set; }
        public int? IdMunicipio { get; set; }
        public int? IdLocalidad { get; set; }
        public int? IdColonia { get; set; }
        public string CodigoPostal { get; set; }
        public string Calle { get; set; }
        public string EntreCalles { get; set; }
        public string NumeroInterior { get; set; }
        public string NumeroExterior { get; set; }

    }
}
